tests/autoinstall_test.go

```diff
@@ -121,9 +121,17 @@ var _ = Describe("kairos autoinstall test", Label("acceptance"), func() {
 			})
 
 			By("checking grubmenu", func() {
-				out, err := vm.Sudo("cat /run/initramfs/cos-state/grubmenu")
+				// Statereset is now part of the default grub.cfg
+				out, err := vm.Sudo("cat /etc/cos/grub.cfg")
 				Expect(err).ToNot(HaveOccurred())
```

Comment:
split into checking the default ids we ship and the optional remoterecovery that can be override by users